apps/web/src/api/hooks/notification-templates/useFetchBlueprints.ts

```diff
@@ -0,0 +1,148 @@
+      },
+    ],
+  },
+  {
+    name: 'Growth',
+    templates: [
+      {
+        id: '4',
+        name: ':fa-regular fa-hand: Welcome message',
+        description:
+          'Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate quas totam quod beatae. Ipsam quasi fugiat commodi adipisci eligendi necessitatibus cumque aliquam, dicta natus cupiditate suscipit voluptatum rerum debitis. Ipsum!',
+      },
+      {
+        id: '5',
+        name: ':fa-solid fa-envelope-open-text: Invite message',
+        description:
+          'Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate quas totam quod beatae. Ipsam quasi fugiat commodi adipisci eligendi necessitatibus cumque aliquam, dicta natus cupiditate suscipit voluptatum rerum debitis. Ipsum!',
+      },
+      {
+        id: '6',
+        name: ':fa-solid fa-gift: Refferal link',
+        description:
+          'Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate quas totam quod beatae. Ipsam quasi fugiat commodi adipisci eligendi necessitatibus cumque aliquam, dicta natus cupiditate suscipit voluptatum rerum debitis. Ipsum!',
+      },
+    ],
+  },
+  {
+    name: 'Authentification',
+    templates: [
+      {
+        id: '7',
+        name: ':fa-solid fa-wand-magic-sparkles: Magic link',
+        description:
+          'Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate quas totam quod beatae. Ipsam quasi fugiat commodi adipisci eligendi necessitatibus cumque aliquam, dicta natus cupiditate suscipit voluptatum rerum debitis. Ipsum!',
+      },
+      {
+        id: '8',
+        name: ':fa-solid fa-unlock: Password change',
+        description:
+          'Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate quas totam quod beatae. Ipsam quasi fugiat commodi adipisci eligendi necessitatibus cumque aliquam, dicta natus cupiditate suscipit voluptatum rerum debitis. Ipsum!',
+      },
+      {
+        id: '9',
+        name: ':fa-solid fa-unlock: Password change2',
+        description:
+          'Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate quas totam quod beatae. Ipsam quasi fugiat commodi adipisci eligendi necessitatibus cumque aliquam, dicta natus cupiditate suscipit voluptatum rerum debitis. Ipsum!',
+      },
+      {
+        id: '10',
+        name: ':fa-solid fa-unlock: Password change3',
+        description:
+          'Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate quas totam quod beatae. Ipsam quasi fugiat commodi adipisci eligendi necessitatibus cumque aliquam, dicta natus cupiditate suscipit voluptatum rerum debitis. Ipsum!',
+      },
+    ],
+  },
+];
+
+const getTemplateDetails = (templateName: string): { name: string; iconName: IconName } => {
+  const regexResult = /^:.{1,}: /.exec(templateName);
```

Comment:
Safe RegExp 👍🏻

Reply:
Also it could work `/^:.{1,}:/`, without the trailing space and therefore the trimming wouldn't be needed later. It is quite subtle but if anyone adds a template with a missing space after the icon placeholder might have a hard time noticing it.

Reply:
That makes sense, I'll update the regexp, but I'll leave the trim just for the case for both.